dataprofiler/tests/profilers/test_unstructured_text_profile.py

```diff
@@ -139,6 +139,32 @@ def test_merge_profiles(self):
         self.assertIn("vocab", profile["times"])
         self.assertIn("words", profile["times"])
         
+    def test_diff_profiles(self):
+        text_profile1 = TextProfiler("Name")
+        sample = pd.Series(["Hello my name is: Grant.!!!"])
+        text_profile1.update(sample)
+
+        text_profile2 = TextProfiler("Name")
+        sample = pd.Series(["Bob and \"Grant\", 'are' friends Grant Grant"])
+        text_profile2.update(sample)
+        
+        expected_diff = {
+            'vocab': [['H', 'l', 'm', 'y', ':', '.', '!'],
+                      ['e', 'o', ' ', 'n', 'a', 'i', 's', 'G', 'r', 't'],
+                      ['B', 'b', 'd', '"', ',', "'", 'f']],
+            'vocab_count': [
+                {'!': 3, 'l': 2, 'm': 2, 'H': 1, 'y': 1, ':': 1, '.': 1},
+                {' ': -2, 'e': 'unchanged', 'n': -3, 'a': -3, 'o': 'unchanged',
+                 'i': 'unchanged', 's': 'unchanged', 'G': -2, 'r': -4, 't': -2},
```

Comment:
should we make `unchanged` as `0`, so the type is consistent with others in this dict?

Reply:
This is consistent with the rest of differences